bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/Rule.java

```diff
@@ -101,6 +104,30 @@ public String getUID() {
     }
 
     /**
+     * Gets the scope of the Rule, if any exist. This property is either set by the {@link RuleEngine} when
+     * the {@link Rule} is added or by the creating party. It's an optional property.
+     * <br/><br/>Implementation note
+     *    <br/>The scope is part of the {@code UID} and the prefix thereof.
+     *    <br/>If the UID does not contain a {@link SCOPE_DELIMITER} {@code null} will be returned.
+     *    <br/>If the UID does contain a {@link SCOPE_DELIMITER} the prefix until the first occurrence will be returned.
+     *    <br/>If the prefix would have a zero length {@code null} will be returned.
+     *
+     * @return scope of this {@link Rule}, or {@code null} if no scope or an empty scope is found
+    */
+    public String getScope() {
```

Comment:
> what do you think about this scope functionality of rule? Is it needed?

My understanding was that we agreed that we do not introduce the notion of "scope", but only implicitly induce it through a potential prefix/namespace within the rule UID.